slider/src/main/java/io/monstarlab/mosaic/slider/SliderTrack.kt

```diff
@@ -0,0 +1,74 @@
+package io.monstarlab.mosaic.slider
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.heightIn
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawBehind
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.geometry.Size
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+
+@Composable
+internal fun SliderTrack(
+    progress: Float,
+    enabled: Boolean,
+    colors: SliderColors,
+    modifier: Modifier = Modifier,
+    disabledRange: ClosedFloatingPointRange<Float> = 0f..0f,
+) {
+    check(progress in 0f..1f) { "Invalid progress value of $progress, should be between 0 and 1" }
```

Comment:
there is a `@FloatRange(..)` for that extra layer of fail-proofing

Reply:
Added :)